lib/release/notes/log.rb

```diff
@@ -42,8 +48,10 @@ def find_last_tag_and_log
       #
       def find_all_tags_and_log_all
         git_all_tags.each_with_index do |ta, i|
-          tag_logger(ta, previous_tag(i))
+          tag_logger(ta, find_previous_tag(i + 1))
         end
+
+        commits_since_last_tag if config_update_release_notes_before_tag?
```

Comment:
I'm torn over doing this here.  Should a rewrite never log the latest? Maybe we add a seperate cmd and force the user to generate a changelog if one doesn't exist

`bin/release-notes create` (if no ouput file exists you must run this cmd and it only creates based on tags that exists)
`bin/release-notes generate` (this only works if a file exists and you must pass a tag option if you want to generate the output before you tag)

Reply:
I agree. I would think a rewrite would log the latest. Is that not what the `force_rewrite` option does?

Reply:
Remind me again the value of allowing a changelog generation prior to generating a tag? I think this is what bothers me the most. I would think we should only generate log messages for tags...

Reply:
Makes sense! I approve.